addons/sourcemod/scripting/WCX_Engine_Crit.sp

```diff
@@ -8,30 +8,30 @@ public Plugin:myinfo =
     description="Generic crit skill"
 };
 
-public OnWar3EventPostHurt(victim, attacker, Float:damage, const String:weapon[32], bool:isWarcraft)
+public int OnWar3EventPostHurt(int victim, int attacker, float damage, const char weapon[32], bool isWarcraft)
```

Comment:
this is source pawn, why are these changed to floats? is this a new sourcemod compiler?